lang/MoarVM/Portfile

```diff
@@ -4,36 +4,27 @@ PortSystem          1.0
 
 PortGroup           compiler_blacklist_versions 1.0
 PortGroup           legacysupport 1.1
-PortGroup           active_variants 1.1
 
 # strnlen, arc4random_buf
 legacysupport.newest_darwin_requires_legacy 10
 
 name                MoarVM
-version             2023.06
-revision            1
+version             2023.12
+revision            0
 categories          lang devel
-license             Artistic-2 MIT BSD ISC public-domain
+license             {Artistic-2 BSD ISC MIT public-domain}
```

Comment:
this change implies one of the license can be used. Are you sure that's the case?

Reply:
@reneeotten No, I am not sure, did not know there is a difference (I had an impression that using these brackets is a codestyle). Sorry.
Should I make a PR to revert this change?

Reply:
See: https://guide.macports.org/#reference.keywords

Yes, please revert this in another PR together with addressing the other comments.